src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -8,6 +8,7 @@
   --pf-c-options-menu__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
   --pf-c-options-menu__toggle--PaddingLeft: var(--pf-global--spacer--sm);
```

Comment:
can you add a `color` to the toggle, similar to my comment about the dropdown, so that the color toggles properly in dark mode?